app/models/contact.rb

```diff
@@ -1,4 +1,32 @@
+require 'services/mailing_list'
 class Contact < ActiveRecord::Base
-  has_many :member_contacts
-  has_many :sponsors, through: :member_contacts
+  belongs_to :sponsor
+
+  validates :name, :surname, :email, presence: true
+
+  before_create :set_token
+  after_commit :subscribe_to_mailinglist, if: proc { |c| c.mailing_list_consent }
+  after_commit :unsubscribe_from_mailinglist, if: proc { |c| !c.mailing_list_consent }
+
+  def subscribe_to_mailinglist
```

Comment:
@despo since mailing list is two words can we rename this method and the one below to `subscribe_to_mailing_list` & `unsubscribe_from_mailing_list` respectively? Also rename the `mailinglist` method to `mailing_list`

Reply:
👍